src/libraries/System.Net.Sockets/ref/System.Net.Sockets.cs

```diff
@@ -99,7 +99,7 @@ public partial class NetworkStream : System.IO.Stream
         public override long Position { get { throw null; } set { } }
         protected bool Readable { get { throw null; } set { } }
         public override int ReadTimeout { get { throw null; } set { } }
-        protected System.Net.Sockets.Socket Socket { get { throw null; } }
+        public Socket Socket { get { throw null; } }
```

Comment:
```suggestion
        public System.Net.Sockets.Socket Socket { get { throw null; } }
```

Reply:
Done. @stephentoub just to understand the suggestion, this is because the property name is the same as the class name?

Reply:
It's actually because we generally use a tool to update these ref files, and it'll use the namespace. So this would just be causing unnecessary churn.